editor/inspector/editor_inspector.cpp

```diff
@@ -1678,7 +1678,8 @@ void EditorInspectorCategory::_notification(int p_what) {
 			if (is_layout_rtl()) {
 				ofs = get_size().width - ofs - w;
 			}
-			float text_pos_y = font->get_ascent(font_size) + (get_size().height - font->get_height(font_size)) / 2 + v_margin_offset;
+			float font_height = font->get_height(font_size);
+			float text_pos_y = (get_size().height - font_height) + (font_height / 2) + v_margin_offset;
```

Comment:
Seems like this can be simplified to this, can you confirm please?
```suggestion
			float text_pos_y = get_size().height - (font_height / 2) + v_margin_offset;
```

Reply:
Good catch, can confirm the simplified version is valid